app/src/main/java/org/oppia/android/app/topic/lessons/TopicLessonsFragmentPresenter.kt

```diff
@@ -180,28 +184,42 @@ class TopicLessonsFragmentPresenter @Inject constructor(
     )
     binding.chapterRecyclerView.adapter = createChapterRecyclerViewAdapter()
 
+    binding.chapterListDropDownIcon.setOnClickListener {
```

Comment:
Because `chapterListDropDownIcon` is now a clickable item it needs to be minimum 48x48 dp.

Maybe instead of using `chapterListDropDownIcon ` we can use `expandListIcon`

Reply:
Updated